### ISSUE TITLE
Get libvpx whitelisted for TravisCI

### ISSUE THREAD

**Issue by tonyc** [curationexperts/mira#447]
Currently, libvpx is not whitelisted for TravisCI's containerized build. This causes any calls to ffmpeg that transcode video to webm to fail, resulting in test failures that execute ffmpeg directly

Follow the instructions at https://github.com/travis-ci/apt-package-whitelist and work on getting libvpx whitelisted if possible

Deliveralbe is a pending pull request to this repo for the libvpx package

**Comment by jcoyne** [comment]
I don't think we should have any tests that call ffmpeg. We should test our code, not libffmpeg. I'd suggest we stub out the system call.
